docs/app/routes/docs/guides/react-three-fiber.mdx

```diff
@@ -0,0 +1,401 @@
+---
+meta:
+  title: R3F | React Spring
+  'og:title': R3F | React Spring
+  'twitter:title': R3F | React Spring
+  description: A detailed guide to using React Spring with R3F.
+  'og:description': A detailed guide to using React Spring with R3F.
+  'twitter:description': A detailed guide to using React Spring with R3F.
+  'og:url': https://beta.react-spring.dev/docs/guides/react-three-fiber
+  'twitter:url': https://beta.react-spring.dev/docs/guides/react-three-fiber
+sidebar_position: 1
```

Comment:
Is there a way to add tags or meta information so this is searchable under "R3F", "react-three-fiber", "three", etc.? You don't want the article title to not have searchable keywords on the site.

Reply:
The search is DocSearch by algolia, I think (when the crawler is ran) it should be searchable by _lots_ of things.